rules/windows/credential_access_lsass_memdump_file_created.toml

```diff
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/11/24"
+maturity = "production"
+updated_date = "2020/11/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of Local Security Authority Process (lsass.exe) default memory dump file name. This may indicate
+credential access attempt via trusted system utilities such as TaskMgr and sqldumper or known pentesting tools such as Dumpert and AndrewSpecial.
```

Comment:
```suggestion
Identifies the creation of a Local Security Authority Subsystem Service (lsass.exe) default memory dump. This may indicate
a credential access attempt via trusted system utilities such as Task Manager (taskmgr.exe) and SQL Dumper (sqldumper.exe) or known pentesting tools such as Dumpert and AndrewSpecial.
```

Reply:
re-lint